addons/hr_recruitment/models/hr_applicant.py

```diff
@@ -107,14 +108,10 @@ class HrApplicant(models.Model):
     attachment_ids = fields.One2many('ir.attachment', 'res_id', domain=[('res_model', '=', 'hr.applicant')], string='Attachments')
     kanban_state = fields.Selection([
         ('normal', 'In Progress'),
-        ('done', 'Ready for Next Stage'),
+        ('done', 'Ready'),
         ('waiting', 'Waiting'),
         ('blocked', 'Blocked')], string='Kanban State',
         copy=False, default='normal', required=True)
-    legend_blocked = fields.Char(related='stage_id.legend_blocked', string='Kanban Blocked')
-    legend_done = fields.Char(related='stage_id.legend_done', string='Kanban Valid')
-    legend_waiting = fields.Char(related='stage_id.legend_waiting', string='Kanban Waiting')
-    legend_normal = fields.Char(related='stage_id.legend_normal', string='Kanban Ongoing')
```

Comment:
So this was "removed as not used and to unify with other views", yet other views such as event tracks and helpdesk keep it?

No entirely sure if it's "not used" specifically in the context of recruitment or if the entire custom kanban labels were not used and should be removed?